app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -151,6 +136,52 @@ class EventDetailsFragment : Fragment() {
         return rootView
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        eventViewModel.event
```

Comment:
Why are you observing live data inside on view ceated?

Reply:
To get the logo url of the image and handle the transition in the Picasso callback otherwise i would need to pass the logo url in the bundle